smallrye-reactive-messaging-mqtt/src/main/java/io/smallrye/reactive/messaging/mqtt/MqttSource.java

```diff
@@ -54,6 +54,7 @@ public MqttSource(Vertx vertx, MqttConnectorIncomingConfiguration config) {
                             }
                             return multi;
                         })
+                        .onOverflow().buffer()
```

Comment:
we should make that configurable.

Reply:
Added a buffer-size config on incoming connector